src/client/components/dimension-list-tile/dimension-list-tile.tsx

```diff
@@ -165,21 +197,36 @@ export class DimensionListTile extends React.Component<DimensionListTileProps, D
 
     var icons: TileHeaderIcon[] = [
       //{ name: 'more', onClick: null, svg: require('../../icons/full-more-mini.svg') }
-      //{ name: 'search', onClick: null, svg: require('../../icons/full-search.svg') }
+      {
+        name: 'search',
+        ref: 'search',
+        onClick: this.toggleSearch.bind(this),
+        svg: require('../../icons/full-search.svg'),
+        active: showSearch
+      }
     ];
 
-    return <div
-      className="dimension-list-tile"
+    const className = classNames(
+      'dimension-list-tile',
+      (showSearch ? 'has-search' : 'no-search')
```

Comment:
this `(showSearch ? 'has-search' : 'no-search')` should ether be removed or moved into SearchableTile